### PR TITLE
[ET-1177] Success/order page

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ET-1177] 

### 🗒️ Description

- Adding templates and styles for the success/order page.
- Adding implementation so that we can demo the complete flow.
- I couldn't set up TC w/Paypal locally, so I ended up working on this like the old days, assuming, pushing to the sandbox, and updating a few items via sFTP.

**Things that we need to revisit**

- After strategy we need to check the naming here, to see if we keep with "success" for the shortcode or we do something more oriented to "order".
- Snapshot tests will come in a later PR.
- See if we can abstract order methods to use them directly (like for method, or total which is not including the currency symbol right now).

### ✔️ Checklist
- [ ] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [ ] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1177]: https://theeventscalendar.atlassian.net/browse/ET-1177

### CODE REVIEW

**Review by bordoni — DISMISSED** [review]

File: src/views/v2/commerce/order/description.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+?>
+<div class="tribe-common-b1 tribe-tickets__commerce-order-description">
+	<?php
+	printf(
+		// Translators: %1$s: Plural `tickets` in lowercase.
+		esc_html__( 'Thank you. Your order has been received. A receipt for purchase and any digital %1$s ordered will be emailed to you shortly.', 'event-tickets' ),
+		tribe_get_ticket_label_plural_lowercase( 'tickets_commerce_order_description' ) // phpcs:ignore
+	);
+	?>
```
Comment (paulmskim) [inline src/views/v2/commerce/order/description.php:32]: this should be escaped last:

```
echo esc_html(
	sprintf(
		// Translators: %1$s: Plural `tickets` in lowercase.
		__( 'Thank you. Your order has been received. A receipt for purchase and any digital %1$s ordered will be emailed to you shortly.', 'event-tickets' ),
		tribe_get_ticket_label_plural_lowercase( 'tickets_commerce_order_description' )
	)
);
```

File: src/views/v2/commerce/order/footer/links/back-home.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+?>
+<a
+	class="tribe-common-anchor-alt tribe-tickets__commerce-order-footer-link tribe-tickets__commerce-order-footer-link--back-home"
+	href="<?php echo esc_url( home_url() ); ?>">
```
Comment (paulmskim) [inline src/views/v2/commerce/order/footer/links/back-home.php:27]:
```
<a
	class="tribe-common-anchor-alt tribe-tickets__commerce-order-footer-link tribe-tickets__commerce-order-footer-link--back-home"
	href="<?php echo esc_url( home_url() ); ?>"
>
```

File: src/views/v2/commerce/order/footer/links/browse-events.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	class="tribe-common-anchor-alt tribe-tickets__commerce-order-footer-link tribe-tickets__commerce-order-footer-link--browse-events"
+	href="<?php echo tribe_events_get_url(); // phpcs:ignore ?>"
+>
+	<?php
+		printf(
+			// Translators: %1$s: Plural `events` in lowercase.
+			esc_html__( 'browse more %1$s', 'event-tickets' ),
+			tribe_get_event_label_plural_lowercase( 'tickets_commerce_order_footer_link' ) // phpcs:ignore
+		);
+		?>
```
Comment (paulmskim) [inline src/views/v2/commerce/order/footer/links/browse-events.php:38]: this should also be late escaped

**Review by paulmskim — COMMENTED** [review]
looking good, just a few small notes

**Pull request merged**